src/test/java/htsjdk/samtools/ValidateSamFileTest.java

```diff
@@ -351,6 +351,22 @@ public void testFatalParsingErrors(final String scenario, final String inputFile
         };
     }
 
+    @Test(dataProvider = "testQualitiesNotStored")
+    public void testNotStoredQualitiesFields(final String inputFile, final double expectedValue) throws IOException {
+        final SamReader reader = SamReaderFactory.makeDefault().open((new File(TEST_DATA_DIR, inputFile)));
```

Comment:
reader should be initialized in a try-with-resources block so that it's closed automatically.

Reply:
Done.